posthog/hogql/database/models.py

```diff
@@ -138,3 +138,12 @@ class FunctionCallTable(Table):
     """
 
     name: str
+
+
+class SavedQuery(Table):
```

Comment:
This is very interesting: are saved queries disjoint from saved HogQL query insights? It would be very powerful if they were one thing, although perhaps there are considerations where HogQL query insights are not yet there as a layer of abstraction instead of the final output.

Reply:
tbh saved hogql query insights would be fine to extend on this but i wasn't sure if trying to extend saved insights into the query stack made as much sense as just having a clear separate system. this way I can freely extend savedqueries into data modeling

Reply:
I guess that's fine. Might be worth to circle back to this at some point though, it's worth making the integration between warehouse and insights seamless